resources/responses/skroutz/merchants_ecommerce_orders_demo_store_pickup.yml

```diff
@@ -0,0 +1,115 @@
+  :verb: :get
+:response:
+  :status: 200
+  :headers:
+    Date: Thu, 24 Jun 2021 15:01:26 GMT
+    Connection: keep-alive
+    Content-Length: 6400
+    Server: h2o (Debian)
+    Content-Type: application/json; charset=utf-8
+    X-Meta-Client-Id: 27951bdf06a65ea4
+    Cache-Control: no-cache
+    X-Request-Id: 989369834
+    X-Runtime: 0.021923
+    X-App: as103_ssl
+    X-Varnish: 989369834
+    Age: 0
+    Via: 1.1 varnish (Varnish/5.0)
+    X-Origin: lb2
+    X-Cache: MISS
+    X-Cached: skroutz miss(0)
+  :body: '{
+  "order": {
+    "code": "DEMO-STORE-PICKUP",
+    "state": "open",
+    "customer": {
+      "id": "user1234",
+      "first_name": "John",
+      "last_name": "Doe",
+      "address": {
+        "street_name": "Πανεπιστημίου",
+        "street_number": "4",
+        "zip": "12345",
+        "city": "Αθήνα",
+        "region": "Αττικής",
+        "country_code": "GR",
+        "pickup_from_collection_point": false
+      }
+    },
+    "invoice": false,
+    "comments": "Παράδοση στο γραφείο",
+    "courier": null,
+    "courier_voucher": null,
+    "courier_tracking_codes": [],
+    "line_items": [
+      {
+        "id": "Y5jVmgKmeX",
+        "shop_uid": "405753",
+        "shop_variation_uid": "40575312-14",
+        "product_name": "Adidas Perormance Badge of Sport Swimsuit PS/GS",
+        "quantity": 2,
+        "size": {
+          "label": "Ηλικία",
+          "value": "14 χρονών",
+          "shop_value": "12-14",
+          "shop_variation_uid": "40575312-14"
+        },
+        "unit_price": 17.99,
+        "total_price": 35.98,
+        "price_includes_vat": true,
+        "ean": "1234567890123",
+        "serial_numbers": null
+      }
+    ],
+    "created_at": "2021-06-24T13:08:30+03:00",
+    "expires_at": "2021-06-25T13:08:30+03:00",
+    "dispatch_until": null,
+    "express": false,
+    "gift_wrap": false,
+    "fulfilled_by_skroutz": false,
+    "fbs_delivery_note": null,
+    "store_pickup": true,
+    "accept_options": {
+      "number_of_parcels": [
+        1
+      ],
+      "pickup_location": [
+        {
+          "id": "Y5jVmgKmeX",
+          "label": "Πανεπιστημίου 2, Τ.Κ. 12345, Αθήνα, Αττική"
+        }
+      ],
+      "pickup_window": []
+    },
+    "reject_options": {
+      "line_item_rejection_reasons": [
+        {
+          "id": 1,
+          "label": "Εκτός αποθέματος στο κατάστημα ή στον προμηθευτή",
+          "requires_available_quantity": false
+        },
+        {
+          "id": 2,
+          "label": "Λάθος καταχωρημένη τιμή(ες)",
+          "requires_available_quantity": false
+        },
+        {
+          "id": 4,
+          "label": "Περιορισμένα τεμάχια",
+          "requires_available_quantity": true
+        },
+        {
+          "id": 5,
+          "label": "To προϊόν καταργήθηκε",
+          "requires_available_quantity": false
+        }
+      ]
+    }
+  }
+}'
```

Comment:
```suggestion
}

```

Reply:
my bad ignore this 
just the new line otherwise it breaks

Reply:
183e898ecd7d7447a4340c5aa92ac9383fc88f7e